src/Accounts/Accounts/Utilities/OpenIDConfiguration.cs

```diff
@@ -0,0 +1,94 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Profile.Properties;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Profile.Utilities
+{
+    internal class OpenIDConfiguration : IOpenIDConfiguration
+    {
+        private const string DefaultAuthority = "https://login.microsoftonline.com/";
```

Comment:
Does this work for regional clouds? Like China cloud.

Reply:
> Does this work for regional clouds? Like China cloud.

Will use ActiveDirectoryAuthority as CLI does